packages/esm-patient-registration-app/src/patient-registration/field/field.test.tsx

```diff
@@ -0,0 +1,282 @@
+              value={{
+                setFieldValue: jest.fn(),
+                setInitialFormValues: jest.fn(),
+                values: {},
+              }}>
+              <Field name="dob" />
+            </PatientRegistrationContext.Provider>
+          </Form>
+        </Formik>
+      </ResourcesContext.Provider>,
+    );
+    expect(screen.getByText('Birth')).toBeInTheDocument();
+  });
+
+  it('should render AddressComponent component when name prop is "address"', () => {
+    jest.mock('./address/address-hierarchy.resource', () => ({
+      ...(jest.requireActual('../address-hierarchy.resource') as jest.Mock),
+      useOrderedAddressHierarchyLevels: jest.fn(),
+    }));
+    (useConfig as jest.Mock).mockImplementation(() => ({
+      fieldConfigurations: {
+        address: {
+          useAddressHierarchy: {
+            enabled: false,
+            useQuickSearch: false,
+            searchAddressByLevel: false,
+          },
+        },
+      },
+    }));
+    render(
+      <ResourcesContext.Provider value={mockResourcesContextValue}>
+        <Formik initialValues={{}} onSubmit={null}>
+          <Form>
+            <PatientRegistrationContext.Provider
+              value={{
+                setFieldValue: jest.fn(),
+                setInitialFormValues: jest.fn(),
+                values: {},
+              }}>
+              <Field name="address" />
+            </PatientRegistrationContext.Provider>
+          </Form>
+        </Formik>
+      </ResourcesContext.Provider>,
+    );
+    expect(screen.getByText('Address')).toBeInTheDocument();
+  });
+
+  it.skip('should render Identifiers component when name prop is "id"', () => {
+    (useConfig as jest.Mock).mockImplementation(() => ({
+      defaultPatientIdentifierTypes: ['OpenMRS ID'],
+    }));
+    render(
+      <ResourcesContext.Provider value={mockResourcesContextValue}>
+        <Formik initialValues={{}} onSubmit={null}>
+          <Form>
+            <PatientRegistrationContext.Provider
+              value={{
+                setFieldValue: jest.fn(),
+                setInitialFormValues: jest.fn(),
+                values: {
+                  patientUuid: '',
+                  givenName: '',
+                  middleName: '',
+                  familyName: '',
+                  unidentifiedPatient: false,
+                  additionalGivenName: '',
+                  additionalMiddleName: '',
+                  additionalFamilyName: '',
+                  addNameInLocalLanguage: false,
+                  gender: '',
+                  birthdate: '',
+                  yearsEstimated: 0,
+                  monthsEstimated: 0,
+                  birthdateEstimated: false,
+                  telephoneNumber: '',
+                  isDead: false,
+                  deathDate: '',
+                  deathCause: '',
+                  relationships: [],
+                  identifiers: { mockedIdentifierTypes },
```

Comment:
Opening this draft PR to get some help on one of the failing test i.e. `should render Identifiers component when name prop is "id"` . The test if failing because of the missing mock value of identifiers. What value should I pass here to get this test Pass ? Please suggest cc / @jayasanka-sack @Piumal1999 @anjula-sack @denniskigen